tmt/utils.py

```diff
@@ -1277,7 +1234,9 @@ def shell_to_dict(variables: Union[str, List[str]]) -> EnvironmentType:
     return result
 
 
-def environment_to_dict(variables: Union[str, List[str]]) -> EnvironmentType:
+def environment_to_dict(logger: tmt.log.Logger,
+                        variables: Union[str,
```

Comment:
we need to do something with this `autopep`, I cannot stop commenting on this, but here really autopep8 sucks :(

Reply:
Fixed in 39891de6dbf9fb4e01471bda84147df2156d9eea